app/javascript/packs/admin_users/user.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Select from 'react-normalized-select';
+import Collapsible from 'react-collapsible';
+
+class User extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+    this.state = {
+      value: '0',
+      editable: false,
+      role: this.props.user.role,
+      status: this.props.user.status,
+      newSalary: false,
+    };
+    this.handleEdit = this.handleEdit.bind(this);
+    this.handleDelete = this.handleDelete.bind(this);
+    this.handleSalaryChange = this.handleSalaryChange.bind(this);
+    this.setNewSalary = this.setNewSalary.bind(this);
+  }
+
+  componentDidMount() {
+    $.ajax({
```

Comment:
this is not a scope of this PR, but we should think about these api requests in future. I don't like that everything is in one pile.